share/doc/homebrew/Formula-Cookbook.md

```diff
@@ -70,7 +70,7 @@ so you can override this with `brew create <url> --set-name <name>`.
 
 ## Fill in the [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)
 
-**We don’t accept formulae without [`homepage`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)s!**
+**We don’t accept formulae without [`homepages`](http://www.rubydoc.info/github/Homebrew/homebrew/master/Formula#homepage%3D-class_method)!**
```

Comment:
A formula has just one `homepage`; a less confusing way to address this is to add an `a` article prior to `homepage` - "We don’t accept formulae without a [`homepage`]"